Tasks/PublishSymbols/Publish-Symbols.ps1

```diff
@@ -165,7 +165,7 @@ function Update-SymbolClient([string]$symbolServiceUri, [string]$symbolClientLoc
     # Check latest package version.
     $availableVersion = Get-SymbolClientVersion $symbolServiceUri
 
-    $symbolPathRoot = Join-Path $env:APPDATA "VSOSymbolClient"
+    $symbolPathRoot = Join-Path $env:AGENT_TOOLSDIRECTORY "VSOSymbolClient"
```

Comment:
>AGENT_TOOLSDIRECTORY [](start = 37, length = 20)

This might not be available in older agents ? (unless you update minimum agent version in task.json),
Suggest you to fall back to APPData in that case